### PR TITLE
PMM-10957 Daily active users

### PR DESCRIPTION
**Pull request by YaroslavPodorvanov**
PMM-10957

Build: SUBMODULES-2959

Better solution use **grafana_stat_daily_active_users**, but it will be available after upgrade Grafana so use alternative solution

https://github.com/grafana/grafana
- [add daily active counts to stats](https://github.com/grafana/grafana/commit/9dfd469afc8bb597c610662612e0fdb13fc2f31f) committed on Sep 13, 2021

Similar metrics in previous PRs:
- grafana_stat_active_users https://github.com/percona/pmm/pull/1123/files

Alternative solution without upgrade Grafana is use GRAFANADB_SELECT from PR https://github.com/percona/pmm/pull/1285

### CODE REVIEW

**Comment by YaroslavPodorvanov** [comment]
Good solution can be available after Grafana upgrade:
```yaml
telemetry:
  - id: GrafanaStatDailyActiveUsers
    source: VM
    query: grafana_stat_daily_active_users
    summary: "Daily active users"
    data:
      - metric_name: "pmm_server_grafana_stat_daily_active_users"
        value: 1
```

So now use alternative solution with **GRAFANADB_SELECT**

**Comment by YaroslavPodorvanov** [comment]
Testing:
```bash
pmm_server_grafana_stat_total_users: 1
pmm_server_grafana_stat_active_users: 0
pmm_server_grafana_stat_daily_active_users: 1
pmm_server_grafana_stat_total_orgs: 1
pmm_server_grafana_stat_totals_annotations: 0
```

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -267,14 +267,22 @@ telemetry:
...
     query: grafana_stat_active_users
-    summary: "Active monthly users"
+    summary: "Monthly active users"
     data:
       - metric_name: "pmm_server_grafana_stat_active_users"
         value: 1
 
+  - id: GrafanaStatDailyActiveUsers
+    source: GRAFANADB_SELECT
+    query: count(*) AS count FROM user WHERE last_seen_at > time('now') - 86400
```
Comment (pkadej) [inline managed/services/telemetry/config.default.yml:280]: What about:
`WHERE date(last_seen_at) = date('now')`?

Comment (YaroslavPodorvanov) [reply managed/services/telemetry/config.default.yml]: So i use same solution as in [Grafana](https://github.com/grafana/grafana/commit/9dfd469afc8bb597c610662612e0fdb13fc2f31f#diff-484a31f371b93925d8bfa3afe2ce1775d22eb911df01aeb1f327cedbb96b4f58R58):
```go
dailyActiveUserDeadlineDate := time.Now().Add(-24*time.Hour)
	sb.Write(`(SELECT COUNT(*) FROM `+dialect.Quote("user")+` WHERE last_seen_at > ?) AS daily_active_users,`, dailyActiveUserDeadlineDate)
```

We send stats each 24 hours so i think in average will be same

**Review by pkadej — APPROVED** [review]

**Review by dliakhov — APPROVED** [review]

**Review by gen1us2k — APPROVED** [review]

**Pull request merged**